src/main/scala/chisel3/util/Cat.scala

```diff
@@ -18,6 +18,7 @@ import chisel3.internal.sourceinfo.SourceInfoTransform
   * Cat(mySeqOfBits)
   * }}}
   */
+@deprecated("Use chisel3.Cat instead", since = "Chisel 3.6.x")
 object Cat {
 
```

Comment:
Please do this as an actual `git mv` instead of copy. You should be able to just make `Cat` an alias in `package object util`. Something like:
```scala
@deprecated("Use chisel3.Cat instead", "Chisel 3.6")
val Cat = chisel3.Cat
```